plugins/modules/ansible_galaxy_install.py

```diff
@@ -113,9 +118,19 @@
     name: community.network:3.0.2
     force: true
 
+- name: Install collection from private git repository
+  community.general.ansible_galaxy_install:
+    token: "{{ token_variable }}"
+    type: collection
+    requirements_file: requirements.yml
+
 """
 
 RETURN = """
+  token:
+    description: The value of the I(token) parameter.
+    type: str
+    returned: always
```

Comment:
Returning the token parameter sounds like a security problem to me. Also it will never actually be returned if `token` is marked as `no_log=True` (which it should be). I would simply remove this return value.